kong/pdk/log.lua

```diff
@@ -640,7 +641,7 @@ do
 
   local function is_valid_value(v, visited)
     local t = type(v)
-    if v == nil or t == "number" or t == "string" or t == "boolean" then
+    if v == nil or v == cjson_null or t == "number" or t == "string" or t == "boolean" then
```

Comment:
not a blocker for this PR, but we should also exclude FFI null from it
```lua
local ffi = require "ffi"

local n = ffi.new("void*")
print(tostring(n))
print(n == nil)

local cjson = require "cjson"

print(cjson.encode(n))
```

Reply:
Is it an extremely unusual case to encounter the FFI null here as the value is typically extracted from request body?